DiscordIntegration/API/Commands/BotCommandSender.cs

```diff
@@ -65,5 +65,10 @@ public override async void RaReply(string text, bool success, bool logToConsole,
 
         /// <inheritdoc cref="CommandSender.Print"/>
         public override async void Print(string text) => await Network.SendAsync(new RemoteCommand(ActionType.SendMessage, ChannelId, text.Substring(text.IndexOf('#') + 1), true));
-    }
+
+		public override bool Available()
+		{
+			throw new System.NotImplementedException();

```

Comment:
huh??

Reply:
I wrote above that I don’t know how to implement this correctly, since I didn’t create this huge pile of code.